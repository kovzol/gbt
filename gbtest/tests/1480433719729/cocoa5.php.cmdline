#!/bin/sh
cd tests/1480433719729
cat cocoa5.php.in | /usr/bin/timeout -k 120 60 /usr/bin/time /home/kovzol/workspace/cocoa-5.1/bin/CoCoAInterpreter-64 > cocoa5.php.out 2>cocoa5.php.err