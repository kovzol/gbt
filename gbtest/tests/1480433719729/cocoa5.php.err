0.00user 0.00system 0:00.00elapsed 100%CPU (0avgtext+0avgdata 10196maxresident)k
0inputs+8outputs (0major+685minor)pagefaults 0swaps
