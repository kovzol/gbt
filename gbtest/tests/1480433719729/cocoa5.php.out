*** Cannot access default package directory: "packages" ***

!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
!!!!! WARNING: Standard CoCoA packages not found !!!!!
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

   ______      ______      ___         ______
  / ____/___  / ____/___  /   |       / ____/
 / /   / __ `/ /   / __ `/ /| |______/___ `  
/ /___/ /_/ / /___/ /_/ / ___ /_____/___/ /  
`____/`____/`____/`____/_/  |_|    /_____/   
With CoCoALib and external libraries CDD FROBBY GFAN GSL NORMALIZ READLINE 
indent(VersionInfo()); -- for information about this version

# use QQ[v8,v7,v4,v3,v2,v1,v5,v6], DegRevLex;

# GBasis(ideal(77-25*v2,v8^2-1*v6^2-1*v5^2+2*v6*v2-1*v2^2+2*v5*v1-1*v1^2,-93-10<1*v6^2-1*v5^2+2*v6*v2-1*v2^2+2*v5*v1-1*v1^2,-93-100                         *v4,3+2*v1,v7^2-1*v6^2-1*<^2+2*v5*v1-1*v1^2,-93-100*v4,3+2*v1,v7^2-1*v6^2-1*v                         5^2+2*v6*v4-1*v4^2+2*v5*v<v4,3+2*v1,v7^2-1*v6^2-1*v5^2+2*v6*v4-1*v4^2+2*v5*v3                         -1*v3^2,-223-100*v3,625-5<^2+2*v6*v4-1*v4^2+2*v5*v3-1*v3^2,-223-100*v3,625-50                         *v8^2+v8^4-50*v7^2-2*v8^2<1*v3^2,-223-100*v3,625-50*v8^2+v8^4-50*v7^2-2*v8^2*                         v7^2+v7^4));
[2*v1 +3, -25*v2 +77, -100*v3 -223, -100*v4 -93, -2446710000*v7^2 +585460000*v5*v6 +1554720000*v6^2 +1156728800*v5 +7560542000*v6 +8810490687, -2446710000*v8^2 +585460000*v5*v6 +1554720000*v6^2 -2415467800*v5 -12062072200*v6 +23242654293, -2446710000*v5^2 +585460000*v5*v6 -891990000*v6^2 -9755597800*v5 +3009661400*v6 -5472912951]

# 